wrapper/src/test/java/integration/refactored/container/ConnectionStringHelper.java

```diff
@@ -39,25 +43,32 @@ public static String getUrl() {
             .getInstances()
             .get(0)
             .getEndpointPort(),
-        TestEnvironment.getCurrent().getInfo().getDatabaseInfo().getDefaultDbName());
+        TestEnvironment.getCurrent().getInfo().getDatabaseInfo().getDefaultDbName(),
+        wrapperPlugins);
   }
 
-  public static String getUrl(String host, int port, String databaseName) {
-    return getUrl(TestEnvironment.getCurrent().getCurrentDriver(), host, port, databaseName);
+  public static String getUrl(String host, int port, String databaseName, String wrapperPlugins) {
+    return getUrl(TestEnvironment.getCurrent().getCurrentDriver(), host, port, databaseName, wrapperPlugins);
   }
 
-  public static String getUrl(TestDriver testDriver, String host, int port, String databaseName) {
+  public static String getUrl(
+      TestDriver testDriver,
+      String host,
+      int port,
+      String databaseName,
+      String wrapperPlugins) {
     final DatabaseEngine databaseEngine = TestEnvironment.getCurrent().getInfo().getRequest().getDatabaseEngine();
     final String requiredParameters = DriverHelper.getDriverRequiredParameters(databaseEngine, testDriver);
-    return DriverHelper.getDriverProtocol(databaseEngine, testDriver)
+    final String url = DriverHelper.getDriverProtocol(databaseEngine, testDriver)
         + host
         + ":"
         + port
         + "/"
         + databaseName
         + requiredParameters
         + (requiredParameters.startsWith("?") ? "&" : "?")
-        + "wrapperPlugins=\"\"";
+        + "wrapperPlugins=";
+    return wrapperPlugins != null ? url + wrapperPlugins : url;
```

Comment:
nit: does it make sense to include "wrapperPlugins=" in the url when the wrapperPlugins variable is null? Should we just leave the property out of the URL in that case? I guess if the tests are still working properly it isn't a big deal

Reply:
Just a reminder that the wrapper has a default list of plugins now. If `wrapperPlugins` is not specified the wrapper will use `auroraConnectionTracker,failover,efm` plugins by default. This change is to remove that list of plugins.